src/lib/custom-resources/cdk-guardduty-admin-setup/runtime/package.json

```diff
@@ -5,15 +5,15 @@
   "main": "dist/index.js",
   "types": "src/index.ts",
   "scripts": {
-    "build": "pnpx esbuild --minify --bundle --platform=node --target=node14 --external:aws-sdk --outfile=./dist/index.js src/index.ts",
```

Comment:
The current Lambda runtime includes nodejs AWS SDK v2.1055.0, but the GuardDuty EKS is introduced in v.2.1063.0. Therefore, this Lambda bundles the SDK library.